File: Codewars/5kyu/going-to-zero-or-to-infinity/Python/solutions1.py
# Python - 3.4.3

from math import floor

def going(n):
    total, x = 0, 1.0
    for i in range(n, 0, -1):
        total += x
        x *= 1.0 / i
    return floor(total * 1e6) / 1e6

File: Codewars/7kyu/figurate-numbers-number-2-pronic-number/C#/solutions1.cs
// C# - 7.3

using System;

public class Kata {
    public static bool IsPronic(int n) {
        if (n < 0) {
            return false;
        }
        int i = (int)Math.Floor(Math.Sqrt(n));
        return (i * (i + 1)) == n;
    }
}
